src/main/core-options.js

```diff
@@ -115,15 +115,15 @@ const options = {
     category: CATEGORY_GLOBAL,
     type: "choice",
     default: [
-      { since: "0.0.10", value: "babylon" },
+      { since: "0.0.10", value: "babel" },
       { since: "1.13.0", value: undefined }
     ],
     description: "Which parser to use.",
     exception: value =>
       typeof value === "string" || typeof value === "function",
     choices: [
       { value: "flow", description: "Flow" },
-      { value: "babylon", description: "JavaScript" },
+      { value: "babel", description: "JavaScript" },
```

Comment:
To keep backward compatibility:

```suggestion
      {
        value: "babylon",
        description: "JavaScript",
        deprecated: "1.16.0",
        redirect: "babel"
      },
      { value: "babel", since: "1.16.0", description: "JavaScript" },
```

And also we should add a test case for `--parser babylon` to make sure it works.